tour-of-go/exports/helpers/helpers_get.go

```diff
@@ -2,7 +2,7 @@ package helpers
 
 var totas int = 45 + 84
 
-func total(bread int, beans int, rice int) int {
+func gettotal(bread int, beans int, rice int) int {
```

Comment:
Fix this

Reply:
Done changing the error from "gettotal" to "getTotal"